package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "redirect",
+  "private": false,
+  "version": "0.0.1",
+  "description": "redirect",
+  "scripts": {
+    "start": "docker run -d --name redirect firstandthird/redirect"
```

Comment:
lets just to node index.js here.  We try to make docker separate from npm scripts